cmd/snap-recovery-chooser/main.go

```diff
@@ -129,29 +129,28 @@ func runUI(cmd *exec.Cmd, sys *ChooserSystems) (rsp *Response, err error) {
 }
 
 func cleanupTriggerMarker() error {
-	err := os.Remove(defaultMarkerFile)
-	if os.IsNotExist(err) {
-		return nil
+	if err := os.Remove(defaultMarkerFile); err != nil && !os.IsNotExist(err) {
+		return err
 	}
-	return err
+	return nil
 }
 
-func chooser(cli *client.Client) error {
-	snappyTesting := os.Getenv("SNAPPY_TESTING_USE_STDOUT") != ""
+func chooser(cli *client.Client) (reboot bool, err error) {
+	snappyTesting := osutil.GetenvBool("SNAPPY_TESTING_USE_STDOUT", false)
```

Comment:
this also ignores the marker, right?

s/SNAPPY_TESTING_USE_STDOUT/SNAPD_CHOOSER_TESTING_DIRECT/ ?

s/snappyTesting/testingMode/ ?

Reply:
We could do something like https://paste.ubuntu.com/p/sXqWJkT7wp/ and check the outputForUI this way (or keeping in a separate test with something like https://paste.ubuntu.com/p/s86HQY8zfn/) - then we don't need this at all (FWIW, I don't like this special test mode as the change in the tests is not that bad IMHO to justify that we keep this in the prod code around). Unless I miss something of course?

Reply:
I think we can drop this now. I took another look at the local testing, and this is not necessary anymore.